### PR TITLE
tree generation updated (prufer code)

### PR DESCRIPTION
**Pull request by mrsac7**
https://www.geeksforgeeks.org/prufer-code-tree-creation/

### CODE REVIEW

File: MyTestGen.sublime-snippet [hunk 1]
```diff
@@ -66,14 +66,38 @@ void strmx(int mxlen, int n) {
...
 
 void tree(int n) {
-	int exc = rand(1,n);
-	for (int i = 1; i <= n; ++i)
-	{
-		if(i==exc) continue;
-		int first = rand(1,n);
-		while(first==i) first = rand(1,n);
-		cout<<first<<" "<<i<<endl;
-	}
```
Comment (mrsac7) [inline MyTestGen.sublime-snippet]: this part of code sometimes generates symmetric pairs i.e (3,7) and (7,3) which is now fixed

File: MyTestGen.sublime-snippet [hunk 2]
```diff
@@ -66,14 +66,38 @@ void strmx(int mxlen, int n) {
...
+	int prufer[n-2];
+    rep(i,0,n){
```
Comment (krashish8) [inline MyTestGen.sublime-snippet]:
```suggestion
    rep(i,0,n-2){
```

Comment (krashish8) [reply MyTestGen.sublime-snippet]: @mrsac7 I just discovered a minor error in your code. Even though the error is small, it produces very unexpected output on my PC.

**Comment by krashish8** [comment]
Anyway, I've fixed it! :smile:

**Pull request merged**